unity3d/Assets/Swrve/SwrveSDK/Messaging/SwrveBaseCampaign.cs

```diff
@@ -67,6 +94,42 @@ public abstract class SwrveBaseCampaign
     }
 
     /// <summary>
+    /// Get the status of the campaign.
+    /// </summary>
+    /// <returns>
+    /// Status of the campaign.
+    /// </returns>
+    public SwrveCampaignState.Status Status {
+        get {
+            return this.State.CurStatus;
+        }
+        set {
+            this.State.CurStatus = value;
+        }
+    }
+
+    /**
+     * Used internally to identify campaigns that have been marked as MessageCenter campaigns on the dashboard.
+     *
+     * @return true if the campaign is an MessageCenter campaign.
+     */
+    public bool IsMessageCenter() {
```

Comment:
Yeah, I'm never 100% sure about these either.  When I'm bringing new stuff in, I often model them on what's in Android, and when Android an "Is", I replicate that on here,

https://github.com/Swrve/swrve-android-sdk/blob/release-4_5/SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveBaseCampaign.java#L101

Is there specific guidelines for those rules?  Are "Is" methods used on Android only because this functionality isn't available?